docs/developer-guide/Server Methods/bundled-json-config-file.md

```diff
@@ -0,0 +1,59 @@
+# Bundled Configuration File
```

Comment:
Can we rename the path to `docs/developer-guide/server-methods/running-basemaps-server-locally.md`.
```suggestion
# Running Basemaps Server Locally
```